### DISCUSSION TITLE
vote: add mungingdata to mrpowers-io

### DISCUSSION THREAD

**Discussion by MrPowers** [mrpowers-io/community#7] [Ideas]
mungingdata.com is currently a Wordpress site.

This proposal is to migrate mungingdata to an open source mrpowers-io/mungingdata repo.

This would allow for other members of the data community to write mungingdata blog posts too.

**Comment by MrPowers** [comment]
+1 (binding)

**Comment by SemyonSinchenko** [comment]
+1 (binding)

**Comment by zeotuan** [comment]
+1 (non binding)
